tests/percy/snapshots.js

```diff
@@ -51,4 +53,18 @@ async function captureVerticalMapSearch (page, percySnapshot) {
   await page.goto(`${TEST_SITE}/locations?query=a`);
   await waitTillHTMLRendered(page)
   await percySnapshot('vertical-map-search');
+}
+
+async function captureVerticalFullPageMapSearch (page, percySnapshot) {
+  await page.goto(`${TEST_SITE}/locations_full_page_map?query=`);
+  await waitTillHTMLRendered(page)
+  await percySnapshot('vertical-full-page-map-list-view');
```

Comment:
For my own edification, it seems like we're taking snapshots of the list view on desktop, map view on mobile, and detail view on mobile. Is that correct?

Reply:
Percy takes snapshots of desktop and mobile by default. So the list view we get mobile and desktop. And the map view and detail view are only mobile. I can split up the list view snapshots to make that a little more clear